dvol/test_dvol.py

```diff
@@ -71,15 +75,64 @@ def test_list_empty_volumes(self):
         dvol.parseOptions(["-p", self.tmpdir.path, "list"])
         self.assertEqual(dvol.voluminous.getOutput(), ["  VOLUME   BRANCH   CONTAINERS "])
 
-    def test_list_multi_volumes(self):
+    # TODO Bring in jml's path_segments strategy.
+    # XXX Fix more boring wrapping output bugs (112 was too large, here).
+    @given(
+            vol_a=text(alphabet=letters[:26], min_size=1, max_size=112/2),
+            vol_b=text(alphabet=letters[:26], min_size=1, max_size=112/2),
+            newbranch=text(alphabet=letters[:26], min_size=1, max_size=112/2),
+        )
+    def test_branch_multi_volumes(self, vol_a, vol_b, newbranch):
+        tmpdir = FilePath(self.mktemp())
+        tmpdir.makedirs()
+
+        assume(vol_a != vol_b)
```

Comment:
Only a suggestion but this my understanding of assume is that it short circuits the test if the condition is not met so should probably go as close to the beginning as possible